src/android/CameraLauncher.java

```diff
@@ -943,6 +943,9 @@ private Bitmap getScaledAndRotatedBitmap(String imageUrl) throws IOException {
             try {
                 fileStream = FileHelper.getInputStreamFromUriString(imageUrl, cordova);
                 image = BitmapFactory.decodeStream(fileStream);
+            }  catch (OutOfMemoryError e) {
```

Comment:
I think it's better to catch any exception and not only `OutOfMemoryError`, it might fail due to other reasons, and then in the error `callbackContext.error` provide the exception message

Reply:
Indeed. `Exception` will catch all exceptions including `OutOfMemoryError`. So do I need to change or you will?

Reply:
@alpesh12 You need to change this before I accept the PR.

Reply:
@infil00p ok will do after testing.

Reply:
@infil00p Now its done can you please merge changes

Reply:
@infil00p i just check travis build failed for iOS can you please help or guide me to successful complete travis test 
Thank you